### PR TITLE
user view all report

### PR DESCRIPTION
**Pull request by truonglocbinh**

### CODE REVIEW

File: app/views/reports/_report.html.erb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<tr>
+  <td><%= report.report_date %></td>
+  <td><%= t "working_durations.#{report.working_duration}" %></td>
+  <td><%= report.lines_code %></td>
+  <td><%= report.created_at.in_time_zone.strftime "%H:%M:%S %d-%m-%Y" %></td>
```
Comment (hellovietnam93) [inline app/views/reports/_report.html.erb]: cho vao I18n

File: app/views/reports/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<div class="row">
+  <div class="breadcrumb-bar">
+  <ul class="list breadcrumb">
```
Comment (vietmt) [inline app/views/reports/index.html.erb]: Indent

**Comment by hellovietnam93** [comment]
Ok

**Comment by vietmt** [comment]
Ok

**Pull request merged**